indico/htdocs/sass/modules/_contributions.scss

```diff
@@ -25,10 +25,10 @@
 
 @mixin picker-root-element-label() {
     .label {
+        @include ellipsis;
+
         float: left;
         max-width: 80%;
-        overflow: hidden !important;
```

Comment:
Have you tested that `@include ellipsis;` correctly overrides whatever was affecting `overflow` to require `hidden !important`?